aws/data_source_aws_neptune_engine_version.go

```diff
@@ -0,0 +1,191 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/neptune"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAwsNeptuneEngineVersion() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsNeptuneEngineVersionRead,
+		Schema: map[string]*schema.Schema{
+			"engine": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "neptune",
+			},
+
+			"engine_description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"exportable_log_types": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Computed: true,
+			},
+
+			"parameter_group_name": {
```

Comment:
Similar note as #15228 here and the rest of this pull request:

```suggestion
			"parameter_group_family": {
```